### DISCUSSION TITLE
Cannot resurrect my Imsai8080 after upgrade to Firmware ver. 1.10

### DISCUSSION THREAD

**Discussion by Radius2023** [thehighnibble/thehighnibble.github.io#55] [Q&A]
Date 05-06-2023
Problems with 1.10 firmware updates to Ver. 1.09
Udo, Dave and experienced users that can help with
this problem. 
 Enjoying my Imsai 8080 under firmware Ver. 1.09
 since my build in 2020 with everything working
as it should. I decided to take the firmware to Ver. 1.10
by dragging and dropping it to the SYS: icon and this
is when the troubles begin. Let me list them and tell you 
the problems I'm having and what I tried to do to correct. 
to no avail.
 1. Drag and drop Imsasim32 ver.1.10 to sys: Icon which
told me was successful. Then rebooted. Followed Dave's
configuration procedure, reset and attempted to apply
power via the power switch but now won't turn on. 

2. Next step was to look at the boot log through Putty terminal
and see that it is in a BOOT LOOP showing no signs of errors.
The Green preamble looks good. 
(See screen shots included here)

![2023-05-05 (2)](https://user-images.githubusercontent.com/132217595/236642461-0f28990a-dc72-4e28-b426-b8983936d530.png)
3. Cannot connect to the webUI as it seems to timeout under Chrome.
 (I can see the imsai8080 icon in available networks and connect with no
internet)
4. Built several 1.10 IMAGE files and made the configurations recommended by Dave
in the latest 2023 firmware build. These will stop the rolling boot but the other.
problems remain. 
![2023-05-05 (3)bootlog](https://user-images.githubusercontent.com/132217595/236642474-03af74af-748f-47a6-bff8-367c6e5e1c4a.png)

**Comment by GrantMeStrength** [comment]
When things go weird, as they did when I was updating my own system, I popped out the ESP32 module and used the Python tools to completely reflash it. I mean, that's an adventure in itself, but it got everything working.

**Reply by udo-munk** [reply]
I am still running 1.09 and have not updated to 1.10 so far, so no experience with this. I can only guess that something with the SD card image is wrong

**Comment by Radius2023** [comment]
________________________________

**Comment by Radius2023** [comment]
05/07/2023
19:52

 John, Udo,

 Your comment is welcome.  At the moment I don't want to take that plunge yet.
I am hoping there is a simple software solution to this riddle. I was looking
forward to what Ver. 1.10/1.11 would bring to the table but now I would gladly
go back to 1.9 and get this box up and running. I would think, in reading Dave's
excellent Firmware ver. 1.10/1.11 drafts, it would be simple to restore the
computer with the latest IMAGE or FIRMWARE and a little configuration. I hope
to hear from Dave about how to approach this.

John
________________________________

**Comment by Radius2023** [comment]
________________________________

**Comment by Radius2023** [comment]
John, Udo,

 Your comment is welcome.  At the moment I don't want to take that plunge yet.
I am hoping there is a simple software solution to this riddle. I was looking
forward to what Ver. 1.10/1.11 would bring to the table but now I would gladly
go back to 1.9 and get this box up and running. I would think, in reading Dave's
excellent Firmware ver. 1.10/1.11 drafts, it would be simple to restore the
computer with the latest IMAGE or FIRMWARE and a little configuration. I hope
to hear from Dave about how to approach this.

John

See 2 more messages
JK


________________________________

**Reply by Radius2023** [reply]
05/08/2023
Afternoon Udo,
 1. Dragged and dropped FIRMWARE Ver. 1.10 to the SYS: Icon on my original micro-sd card that came with the build
 2. back in 2019. Powered down and rebooted. This was the beginning of my problems.
 3. On 05/06/23 decided to build a new Ver. 1.10 on a new micro-sd card, creating ROM and Tape folders, moving *.hex files
 Editing the /imsai/conf/sys file to include memory configuration sections.

05/08/2023
Making progress. Built a new Ver. 1.10 on new/formatted sd card.  See screenshots below. 
Now can boot CPM 2.2 with an A> prompt on Putty but no KB. Also no WebUI,. Although is shows
up in my available nets as imsai 8080 and connects with "no internet".
See new screenshot's below. I am just about there but need some help to get back up. hope the screenshots
will help. 

John
![2023-05-08 (1)](https://user-images.githubusercontent.com/132217595/236952792-3b938b6c-56f4-4a18-ac2c-e14389934fe5.png)
![2023-05-08 (2)](https://user-images.githubusercontent.com/132217595/236952807-7f963b6b-e711-4e5d-88b0-2d460eb2e06f.png)
![2023-05-08 (3)](https://user-images.githubusercontent.com/132217595/236952836-41c09dc5-4fa1-4ac2-bd29-4ab5a12988f0.png)
![2023-05-08 (4)](https://user-images.githubusercontent.com/132217595/236952861-22d57c3f-9022-4549-8946-a9f5c765a91b.png)
![2023-05-08 (5)](https://user-images.githubusercontent.com/132217595/236952874-7721e1b2-a361-42f4-a738-60e26f7e02d5.png)

**Comment by udo-munk** [comment]
If you still have the SD card that works with the 1.9 software there should be no problem to downgrade again to this version. The advice was to create a complete new SD card for 1.10 and later because of incompatible changes, did you do this or tried to use the old card?

**Comment by nbreeden2** [comment]
See my comments in #66 where I explain this.
